tests/Api/Admin/CustomerTest.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Api\Admin;
+
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Customer\Model\CustomerGroupInterface;
+use Sylius\Tests\Api\JsonApiTestCase;
+use Sylius\Tests\Api\Utils\AdminUserLoginTrait;
+use Symfony\Component\HttpFoundation\Response;
+
+final class CustomerTest extends JsonApiTestCase
```

Comment:
```suggestion
final class CustomersTest extends JsonApiTestCase
```
to be consistent with other classes